hassio/src/system/hassio-host-info.js

```diff
@@ -109,16 +116,23 @@ class HassioHostInfo extends EventsMixin(PolymerElement) {
     }
   }
 
-  computeUpdateAvailable(data) {
-    return data.version !== data.last_version;
+  _dataChanged(data) {
+    if (data.features && data.features.includes('hassos')) {
+      this.hass.callApi('get', 'hassio/hassos/info')
```

Comment:
You can use async/await 👍 (after you rebase so you get the compatibility fix I just merged)

```js
async _dataChanged(data) {
  if (!data.features || !data.features.includes('hassos') return;

  const resp = await this.hass.callApi('get', 'hassio/hassos/info');
  this._hassOs = resp.data;
}
```

Reply:
thanks, Pascal still has to test this PR as I don't have a system running HassOS